great_expectations/expectations/expectation.py

```diff
@@ -1367,6 +1367,188 @@ def _validate(
         )
 
 
+class MulticolumnMapExpectation(TableExpectation, ABC):
+    map_metric = None
+    domain_keys = (
+        "batch_id",
+        "table",
+        "column_list",
+        "row_condition",
+        "condition_parser",
+    )
+    success_keys = tuple()
```

Comment:
Shouldn't we support mostly?

Reply:
@jcampbell In my opinion, `mostly` would belong in a separate PR, and I think that `mostly` does make sense for multicolumn expectations.

Reply:
Works for me.